src/pages/CreateAssignment/CreateAssignment.js

```diff
@@ -77,22 +96,30 @@ function CreateAssignment({ firebase }) {
     setShowWriting(true);
   }
   function verifyNameAndPush() {
+    var options = { month: "long" };
+    let month = new Intl.DateTimeFormat("en-US", options).format(date);
+    let day = date.getDate();
+    let year = date.getFullYear();
+    let defaultName = `${wordGroup}: ${month} ${day} ${year}`;
+
+    // regex expression to check if lessonName is defaultName format by checking if last four chars are numbers (year format)
+    // TODO: better check defaultName
+    const defaultNameRegExp = /.([0-9]{4})$/;
```

Comment:
i feel like whether you should even do this or not is more of a product question @rxun , also how are you gonna leave a TODO in here and call it done

Reply:
Ahh ok good point: I just did the defaultName based off the current format in the code.

The reason why I made this a TODO was because it's functional as of the moment and would take significantly more effort to flush out. Basically what it's doing is checking if the current name is in the format: "[WordGroup]: [Month] [Day] [Year]" which is incredible complex to check even with regex, so the simple solution right now is just to check if the last four letters indicate a year (are numbers). Definitely better ways to do it but should be feasible for the common case for now